fedimint-testing/src/ln.rs

```diff
@@ -258,4 +258,8 @@ impl ILnRpcClient for FakeLightningTest {
     async fn list_active_channels(&self) -> Result<Vec<ChannelInfo>, LightningRpcError> {
         unimplemented!("FakeLightningTest does not support listing active channels")
     }
+
+    async fn get_balances(&self) -> Result<GetBalancesResponse, LightningRpcError> {
+        unimplemented!("FakeLightningTest does not support getting balances")
```

Comment:
`unimplemented!` will panic, no? We probably would rather return an error here and existing places.

Reply:
We're already doing unimplemented in the others, refactoring to returning LightningRpcError for the other unimplemented ones can be a follow up PR

Reply:
Done in #5859